ERCS/erc-7092.md

```diff
@@ -165,162 +135,289 @@ interface IERC7092 {
+    * @param _from array of bondholder accounts
+    * @param _to array of accounts to transfer bond tokens to
+    * @param _amount array of amounts of bond tokens to transfer.
+    * @param _data array of additional information provided by the token holder
+    *
+    ** OPTIONAL - interfaces and other contracts MUST NOT expect this function to be present.
     */
-    function transferFrom(address _from, address _to, uint256 _amount, bytes calldata _data) external returns(bool);
+    function batchTransferFrom(address[] calldata _from, address[] calldata _to, uint256[] calldata _amount, bytes[] calldata _data) external returns(bool);
+}
+```
+
+### Additional bond parameters Interface
+
+The `IERC7092ESG` interface is OPTIONAL for contracts implementing this proposal. This interface MAY be used to improve the standard usability.
+
+- The `currencyOfCoupon` The currency used for coupon payment may be different from the currency used to repay the principal
+- The `couponType` MAY be employed to signify the interest rate that the issuer has committed to paying to investors, which may take various forms such as zero coupon, fixed rate, floating rate, and more.
+- The `couponFrequency` refers to how often the bond pays interest to its bondholders, and is typically expressed in terms of time periods, such as: Annual, Semi-Annual, Quarterly, or Monthly.
+- The `dayCountBasis` is used to calculate the accrued interest on a bond between two coupon payment dates or other specific periods. Some of the day count basis are: Actual/Actual, 30/360, Actual/360, Actual/365, or 30/365
 
+```solidity
+// SPDX-License-Identifier: CC0-1.0
+pragma solidity ^0.8.0;
+
+interface IERC7092ESG /** is ERC165 */ {
     /**
-    * @notice Moves all bonds from `_from` to `_to`. The caller must have been authorized through the approve function
-    * @param _from the bondholder address
-    * @param _to the address to transfer bonds to
-    * @param _data additional information provided by the token holder
-    *
-    * OPTIONAL - interfaces and other contracts MUST NOT expect these values to be present. The method is used to improve usability.
+    * @notice Returns the number of decimals used by the bond. For example, if it returns `10`, it means that the token amount MUST be multiplied by 10000000000 to get the standard representation.
     */
-    function transferAllFrom(address _from, address _to, bytes calldata _data) external returns(bool);
+    function decimals() external view returns(uint8);
 
     /**
-    * @notice MUST be emitted when bonds are transferred
-    * @param _from the account that owns bonds
-    * @param _to the account that receives the bond
-    * @param _amount the amount of bonds to be transferred
-    * @param _data additional information provided by the token holder
+    * @notice Rreturns the coupon currency, which is represented by the contract address of the token used to pay coupons. It can be the same as the one used for the principal
```

Comment:
```suggestion
    * @notice Returns the coupon currency, which is represented by the contract address of the token used to pay coupons. It can be the same as the one used for the principal
```